pom.xml

```diff
@@ -157,7 +153,7 @@
     <maven-surefire.version>2.22.2</maven-surefire.version>
     <metrics.version>4.1.0</metrics.version>
     <mockito.version>3.0.0</mockito.version>
-    <mssql.version>7.4.1.jre8</mssql.version>
+    <mssql.version>7.4.1.jre11</mssql.version>
```

Comment:
If we want to keep the profiles we could override this for if java11 is detected?